### PR TITLE
LPS-186235 | Test Automation | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-186234](https://issues.liferay.com/browse/LPS-186234)
[LPS-186235](https://issues.liferay.com/browse/LPS-186235)
[LPS-186236](https://issues.liferay.com/browse/LPS-186236)

### CODE REVIEW

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldEncryption.testcase [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+				fragmentName = "Collection Display",
+				infoListProviderName = "Custom Objects 186236",
+				styleDisplay = "Table");
+
+			Button.clickPublish();
+		}
+
+		task ("When go to the created page") {
+			Navigator.gotoPage(pageName = "Blank Display Page");
+		}
```
Comment (victorhcunha) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldEncryption.testcase:164]: Task message modified with help from @JhossephG

Comment (victorhcunha) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldEncryption.testcase:164]: Modified task message with help from @JhossephG

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by timpak** [comment]
Unfortunately, we'll need to re-think our approach. I'm investigating setting up a build script to add the properties before we start portal. This may mean that we'll need to remove the feature flag before merging our tests.

**Comment by evanilson-santana** [comment]
Hey @timpak since this PR was opened a few months ago and we won't work on it in the next few days, I'm going to close it.

cc: @victorhcunha

**Pull request closed without merging**